plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -212,7 +212,13 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
                EOS_ASSERT(false, plugin_exception, "");
             }
 #endif
-         }), "Override default WASM runtime")
+         }), "Override default WASM runtime(\"eos-vm\", \"eos-vm-jit\", \"eos-vm-oc\").\n"
+         "\"eos-vm\" : A WebAssembly interpreter providing deterministic and efficient time bound execution with debugging support.\n"
+         "\"eos-vm-jit\" : A WebAssembly compiler that takes WASM and generates native code on the fly.\n"
+         "\"eos-vm-oc\" : A specialized compiler framework (LLVM) that leverages a multi-pass compilation architecture."
+		 "The resulting native code from the Optimized Compiler is often an order of magnitude faster thatn others.\n"
+		 "If EOSIO_EOS_VM_OC_DEVELOPER is not defined, enable 'eos-vm-oc-enable' option instead."
```

Comment:
`eos-vm-oc` should not be listed unless `EOSIO_EOS_VM_OC_DEVELOPER` is enabled.
Also I would remove the sentence "If EOSIO_EOS...".

Reply:
Also, there is a `EOSIO_EOS_VM_RUNTIME_ENABLED` and a `EOSIO_EOS_VM_JIT_RUNTIME_ENABLED`. With the recent removal of wabt, I guess we pretty much assume at least one of these need to be set. I'd still gate the eos-vm-jit  text off `EOSIO_EOS_VM_JIT_RUNTIME_ENABLED` though.

Reply:
Added #ifdef for the EOS_VM_OC and EOS_VM_JIT

  --wasm-runtime runtime                Override default WASM runtime("eos-vm",
                                        "eos-vm-jit")
                                        "eos-vm" : (Default runtime) A 
                                        WebAssembly interpreter providing 
                                        deterministic and efficient time bound 
                                        execution with debugging support.
                                        "eos-vm-jit" : A WebAssembly compiler 
                                        that takes WASM and generates native 
                                        code using a just in time compiler.


  --wasm-runtime runtime                Override default WASM runtime("eos-vm",
                                        "eos-vm-jit", "eos-vm-oc")
                                        "eos-vm" : (Default runtime) A 
                                        WebAssembly interpreter providing 
                                        deterministic and efficient time bound 
                                        execution with debugging support.
                                        "eos-vm-jit" : A WebAssembly compiler 
                                        that takes WASM and generates native 
                                        code using a just in time compiler.
                                        "eos-vm-oc" : A specialized compiler 
                                        framework (LLVM) that leverages a 
                                        multi-pass compilation architecture.
                                        The resulting native code from the 
                                        Optimized Compiler is often an order of
                                        magnitude faster thatn others.